src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+3.5.0
```

Comment:
It will be `4.1.0` not `3.5.0` (which never will be released)

Reply:
ok